README.md

```diff
@@ -37,7 +37,7 @@ To execute end-to-end (e2e) tests, use:
     grunt package
     grunt test:e2e
 
-To learn more about the grunt tasks, run `grunt --help` and also read our
+To learn more about the grunt tasks, run `grunt --help`, instead you may want to read our
```

Comment:
This is a bit clunky. Reading the contributing guidelines has nothing to do with running grunt --help. The second part should be split from the sentence, starting with something like "If you want to contribute, you may want to read ..."